libraries/c_sdk/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -666,10 +670,10 @@ static void _sendPuback( _mqttConnection_t * pMqttConnection,
     size_t pubackSize = 0, bytesSent = 0;
 
     /* Default PUBACK serializer and free packet functions. */
-    IotMqttError_t ( * serializePuback )( uint16_t,
-                                          uint8_t **,
-                                          size_t * ) = _IotMqtt_SerializePuback;
-    void ( * freePacket )( uint8_t * ) = _IotMqtt_FreePacket;
+    IotMqttError_t (* serializePuback)( uint16_t,
+                                        uint8_t **,
+                                        size_t * ) = _pubackSerializeWrapper;
+    void (* freePacket)( uint8_t * ) = _IotMqtt_FreePacket;
```

Comment:
Another example of code style diff. Please run uncrustify, it should reduce diff as well.